llvm/include/llvm/Demangle/Demangle.h

```diff
@@ -32,7 +32,7 @@ enum : int {
 /// Returns a non-NULL pointer to a NUL-terminated C style string
 /// that should be explicitly freed, if successful. Otherwise, may return
 /// nullptr if mangled_name is not a valid mangling or is nullptr.
-char *itaniumDemangle(std::string_view mangled_name);
+char *itaniumDemangle(std::string_view mangled_name, bool ParseParams = true);
```

Comment:
A comment near the top of this file says that this is a copy of __cxa_demangle, which implies that you'll need to make the same changes to that file as you're making to this one. I believe this also applies to the .cpp.

Reply:
> This is a llvm local version of __cxa_demangle. Other than the name and being in the llvm namespace it is identical.

Looks like that comment is outdated. I do not see any relevant place in libcxxabi other than llvm-project/libcxxabi/src/cxa_demangle.cpp.
But llvm-project/llvm/include/llvm/Demangle/Demangle.h (and llvm-project/llvm/lib/Demangle/Demangle.cpp) is absolutely different.